### ISSUE TITLE
Add titles to API-delegation and Single-rights pages

### ISSUE THREAD

**Issue by allinox** [Altinn/altinn-access-management-frontend#787]
![Screenshot 2024-03-12 at 13 57 44](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/18312a63-fb49-478b-b44f-18ab8ec7b8f7)

WCAG requires all pages to have titles (displayed in the browser's tab) and though our pages do have them, they are all the same: Altinn. These titles should instead be more connected to the content/process currently being performed.

Suggested change is:
- All API-deleg pages have title "API delegation - Altinn" 
- All Single-rights pages should have title "Single rights delegation - Altinn"

These titles should probably change to match the current language
